### PR TITLE
Fix bug where dropdown content was positioned in incorrectly on scrolling for first time

### PR DESCRIPTION
**Pull request by Ghusar**

### CODE REVIEW

**Comment by ankit-m** [comment]
@Ghusar build is failing

**Comment by Ghusar** [comment]
Should i commit package-lock.json also for resolving this build?

File: src/Dropdown.js [hunk 1]
```diff
@@ -49,12 +49,30 @@ export default class Dropdown extends Component {
...
+      currentTriggerBoundingRect.left !== updatedTriggerBoudingRect.left ||
+      currentTriggerBoundingRect.top !== updatedTriggerBoudingRect.top ||
+      currentTriggerBoundingRect.left !== updatedTriggerBoudingRect.left ||
+      currentTriggerBoundingRect.right !== updatedTriggerBoudingRect.right ||
+      currentTriggerBoundingRect.height !== updatedTriggerBoudingRect.height
+    );
+
+    if (hasTriggerBoundingRectUpdated) {
+      this.setState({
+        triggerBoundingRect: updatedTriggerBoudingRect
```
Comment (ankit-m) [inline src/Dropdown.js]: No  need for new line

File: src/Dropdown.js [hunk 2]
```diff
@@ -49,12 +49,30 @@ export default class Dropdown extends Component {
...
 
   setTriggerRect() {
-    this.setState({
-      triggerBoundingRect: getAbsoluteBoundingRect(this.triggerRef.current),
-    });
+    if (!this.triggerRef.current) {
+      return;
+    }
+
+    const currentTriggerBoundingRect = this.state.triggerBoundingRect || {};
```
Comment (ankit-m) [inline src/Dropdown.js]: You can simply set trigger rect in handler for opening the dropdown.

**Review by ankit-m — CHANGES_REQUESTED** [review]

**Review by ankit-m — APPROVED** [review]

**Pull request merged**